web/views/scheduled.erb

```diff
@@ -48,7 +48,23 @@
   </div>
 </div>
 <div class="row">
-  <div class="small-12 columns">
+  <div class="small-8 columns">
     <%== erb :_pager %>
   </div>
+  <% if @list.total > 0 %>
+    <div class="small-4 columns text-right">
+      <form action="<%= link_to "jobs" %>" method="post" class="form-inline">
+        <input type="hidden" name="_method" value="put" />
+        <input type="hidden" name="scope" value="scheduled" />
+        <button class="button small" title="Run All Immediately"><i class="fa fa-play-circle"></i> Run All</button>
+      </form>
+      &nbsp;
+      <form action="<%= link_to "jobs" %>" method="post" class="form-inline"
+            onsubmit="return confirm('Are you sure you want to delete all <%= @list.total %> jobs?');">
+        <input type="hidden" name="_method" value="delete" />
+        <input type="hidden" name="scope" value="scheduled" />
+        <button class="button small alert" title="Delete All"><i class="fa fa-trash"></i> Delete All</button>
+      </form>
+    </div>
+  <% end %>
```

Comment:
This duplicates the delete/retry buttons of `failing.erb`. Create a shared template and call it via `<%== erb :_all_job_actions %>`

Reply:
A partial template was the original plan, but the icons and action names are actually not the same (`Retry All` vs `Run All`). 

Compare `failing.erb`
![image](https://user-images.githubusercontent.com/1374745/35589540-833fdc3a-0604-11e8-9a3c-ca7465065ceb.png)

and `scheduled.erb`
![image](https://user-images.githubusercontent.com/1374745/35589556-935ada52-0604-11e8-8364-4f7328013ead.png)

Do you want to unify this somehow?